working-groups.md

```diff
@@ -0,0 +1,65 @@
+# Working Groups 
+
+### [Business Planning][biz-wg]
+
+WG for performing feasibility study, and business planning to identify start up costs, fundraising mechanisms, and risks.
+
+📅 Weekly calls  
```

Comment:
This seems duplicate info as calendar and will fall out of sync

Reply:
I think adding the date would create that issue, but I'm not seeing "weekly" as a major concern, as we've kept to a pretty regular meeting schedule since each group has started? 

Could have these link to the calendar all anyway so that remains the primary source?

Reply:
Will add link to calendar as a way to resolve this.